audio.py

```diff
@@ -77,11 +77,11 @@ def _build_mel_basis():
 
 
 def _amp_to_db(x):
-    return 20 * np.log10(x + 0.01)
+    return 20 * np.log10(np.maximum(1e-5, x))
```

Comment:
You can also use `librosa.amplitude_to_db` for convenience — https://librosa.github.io/librosa/_modules/librosa/core/spectrum.html#amplitude_to_db
(and the same for _db_to_amp)

Also, 0.01 equals to -40 dB, and 1-e5 equals to -100 dB. -40 dB is almost the silence already (values below is only a mic noise usually), do you really want to keep it up to -100 dB?

Reply:
Thanks for the comments. The purpose of the PR is to rollback to the previous code, so I would want to keep the limit to -100dB, since it did worked before. As @rafaelvalle says, the threshold is data dependent and users can always change the value so I think OK to keep the default to -100dB. If you find -40dB do work better in practice, PR is appreciated.

Reply:
Yes, sure, I already set the model training with -40dB, so I'll return to you with pre-trained model & PR if it'll have a good quality.